src/NHapi.Base/Parser/XMLParser.cs

```diff
@@ -83,6 +86,12 @@ protected XMLParser(IModelClassFactory factory)
             };
         }
 
+        public bool IncludeLongNameInEncodedXML
```

Comment:
I would prefer the use of `ParserOptions` to do this, I know you are emulating the other settings on this file but, nhapi is moving towards the use of a configuration object.

If you look at the hapi version of [XMLParser](https://github.com/hapifhir/hapi-hl7v2/blob/3333e3aeae60afb7493f6570456e6280c0e16c0b/hapi-base/src/main/java/ca/uhn/hl7v2/parser/XMLParser.java#L138) you will see these static properties have been deprecated in favour of the configuration object approach also.